examples/using-remark/package.json

```diff
@@ -19,7 +19,7 @@
     "gatsby-remark-images": "latest",
     "gatsby-remark-katex": "latest",
     "gatsby-remark-prismjs": "latest",
-    "gatsby-remark-responsive-iframe": "latest",
+    "gatsby-remark-responsive-iframe": "^1.4.9",
```

Comment:
Was this change related to the issue, or is it something else?

Reply:
No, I just ran yarn upgrade in this example site before using it to test your PR